crypto/build_dependency.sh

```diff
@@ -5,6 +5,26 @@ set -euo pipefail
 PKG_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 RELIC_DIR_NAME="relic"
 RELIC_DIR="${PKG_DIR}/${RELIC_DIR_NAME}"
+MOD_DIR="/pkg/mod/"
```

Comment:
This is maybe still wip, so just an early comment: 
This new part of the script should live outside the crypto package, in our case it would be under the root `flow-go/`. The reason is that `go get` copies the entire crypto package files under `GOPATH/../crypto@v0...`. There needs to be a new script that is run from the project root, and is able to find that specific GOPATH folder.

Reply:
I can continue working on this PR next week if you prefer, since I already did something similar in a local branch.

Reply:
Currently, this script will perform both the install in the mod folder, and for the one it currently lives in, and is my preferred approach. There is no reason that it has to be a separate script in my opinion, but we can chat about it if you like.

Reply:
I didn't fully understand the approach I guess, so I'll wait till the PR is ready.